ui.tests/test-module/specs/ruleeditor/ruleEditor.authoring.spec.js

```diff
@@ -0,0 +1,135 @@
+const commons = require('../../libs/commons/commons'),
+    sitesSelectors = require('../../libs/commons/sitesSelectors'),
+    formsSelectors = require('../../libs/commons/guideSelectors'),
+    afConstants = require('../../libs/commons/formsConstants');
+
+describe('Rule editor sanity for core-components',function(){
```

Comment:
Can we also add these test cases in SITE page, you can refer other test cases, we follow this pattern everywhere

Reply:
Done.